api/HTMLBodyElement.json

```diff
@@ -1,8 +1,8 @@
 {
   "api": {
-    "HTMLBodyElement": {
+    "HTMLEmbdElement": {
```

Comment:
I guess these changes to HTMLBodyElement -> HTMLEmbdElement were just for testing purposes? Can you undo this?